### PR TITLE
feat: atomic sum on v8 values

### PR DESCRIPTION
**Pull request by losfair**

### CODE REVIEW

**Comment by johnspurlock-skymethod** [comment]
How does this handle Number.NaN, Number.NEGATIVE_INFINITY, Number.POSITIVE_INFINITY ?

**Comment by losfair** [comment]
> How does this handle Number.NaN, Number.NEGATIVE_INFINITY, Number.POSITIVE_INFINITY ?

Same as in JavaScript, because both V8 and Rust implement IEEE-754 floating point numbers.

File: sqlite/backend.rs [hunk 1]
```diff
@@ -312,10 +318,32 @@ impl SqliteBackend {
...
+          } => {
+            if matches!(operand, KvValue::U64(_)) {
+              mutate_le64(
+                tx,
+                &mutation.key,
+                "sum",
+                operand,
+                version,
+                |a, b| a.wrapping_add(b),
+              )?;
```
Comment (igorzi) [inline sqlite/backend.rs:335]: How can this code path be triggered from JS? By using `Deno.AtomicOperation.mutate()` method, and passing `KvU64` value?

Comment (losfair) [reply sqlite/backend.rs:335]: Yes

**Review by igorzi — APPROVED** [review]

**Pull request merged**